tests/strategy_tests.py

```diff
@@ -141,76 +140,72 @@ def test_get_form_data(self):
         }
         self.assertEqual(result, expected)
 
-    def test_dummy_strategy(self):
-        strategy = DummyStrategy()
-        result = sorted(strategy.get_urls())
-        expected = [
-            f"{TEST_URL}/?form_data=%7B%27slice_id%27%3A+1%7D",
-            f"{TEST_URL}/?form_data=%7B%27slice_id%27%3A+17%7D",
-            f"{TEST_URL}/?form_data=%7B%27slice_id%27%3A+18%7D",
-            f"{TEST_URL}/?form_data=%7B%27slice_id%27%3A+19%7D",
-            f"{TEST_URL}/?form_data=%7B%27slice_id%27%3A+30%7D",
-            f"{TEST_URL}/?form_data=%7B%27slice_id%27%3A+31%7D",
-            f"{TEST_URL}/?form_data=%7B%27slice_id%27%3A+8%7D",
-        ]
-        self.assertEqual(result, expected)
-
     def test_top_n_dashboards_strategy(self):
         # create a top visited dashboard
         db.session.query(Log).delete()
         self.login(username="admin")
+        dash = self.get_dash_by_slug("births")
         for _ in range(10):
-            self.client.get("/superset/dashboard/3/")
```

Comment:
Non deterministic use of ids, we're lucky that even worked at all... Moving to a deterministic approach